pyqt-apps/siriushla/si_di_bbb/main.py

```diff
@@ -13,6 +13,7 @@
 from .bbb import BbBMainSettingsWidget
 from .util import get_bbb_icon
 from .gpio import BbBGPIOWidget
+from .util import set_bbb_color
```

Comment:
is this necessary?

Reply:
I don't think so. I will remove it! Thanks!